make/test.mk

```diff
@@ -32,13 +32,41 @@ SETUP_E2E_SERVICE_ACCOUNTS ?= true
 
 .PHONY: test-e2e
 ## Run the e2e tests
-test-e2e: deploy-e2e e2e-run
+test-e2e: INSTALL_OPERATOR=true
+test-e2e: prepare-e2e cover-migration-and-deploy e2e-run
 	@echo "The tests successfully finished"
 	@echo "To clean the cluster run 'make clean-e2e-resources'"
 
+.PHONY: test-e2e-without-migration
+## Run the e2e tests with migration tests
+test-e2e-without-migration: prepare-e2e deploy-e2e e2e-run
+	@echo "To clean the cluster run 'make clean-e2e-resources'"
+
+.PHONY: cover-migration-and-deploy
```

Comment:
"cover" may be a bit misleading in the context of tests: it may suggest that there will be some code coverage (which is not the case, AFAICS)

Reply:
is this better? 8a83801